src/lib/validator.js

```diff
@@ -18,7 +18,7 @@
  * Gates Foundation
  - Name Surname <name.surname@gatesfoundation.com>
 
- - Georgi Georgiev <georgi.georgiev@modusbox.com>
+ * Georgi Georgiev <georgi.georgiev@modusbox.com> : sourced from ml-api-adapter
```

Comment:
`sourced from ml-api-adapter`?

Reply:
Replaced by: `<< central-services-shared`. Let's have this label until we decide whether it will be moved to central-services-shared or not. Then we should either delete the script or the comment.